bin/nv

```diff
@@ -18,14 +18,21 @@ require('yargs')
         defaultDescription: 'pretty print json spaces, default 2 (--no-pretty-json for new line delimted json)',
         description: 'Pretty print json'
       })
+      yargs.option('only-version', {
+        alias: 'v',
```

Comment:
I usually hesitate to add a `-v` alias for anything other than `--verbose`. Are you open to not having the alias?

Reply:
sure, that's fine (i usually see it as short for `--version`, actually, which is a stronger argument to omit it)